core/storage/user/gae_models.py

```diff
@@ -148,6 +148,20 @@ class UserStatsModel(base_models.BaseMapReduceBatchResultsModel):
     # The impact score.
     impact_score = ndb.FloatProperty(indexed=True)
 
+class UserDashboardStatsModel(base_models.BaseMapReduceBatchResultsModel):
```

Comment:
I lean towards reusing, actually. Everything in both models is updated using batch jobs, so it seems reasonable to bring them all together. The impact score isn't large so I think there's no need to worry about retrieving too much unneeded info, and the UIs of the dashboard and profile page may well change in the future to highlight more of these stats.